Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -7,6 +7,10 @@ import UIKit
 
 var displayingForm = false
 
+protocol SentryUserFeedbackWidgetDelegate: NSObjectProtocol {
```

Comment:
l: If this is a `NSObjectProtocol` should it also have `@objc`?

Reply:
correct, it doesn't need it, it's implied